xbmc/GUIUserMessages.h

```diff
@@ -103,6 +103,10 @@
 // Message sent to tell the GUI to update a single item
 #define GUI_MSG_UPDATE_ITEM           GUI_MSG_USER + 29
 
+  // Flags for GUI_MSG_UPDATE_ITEM message
+  #define GUI_MSG_FLAG_UPDATE_LIST    0x00000001
```

Comment:
I would prefer using a const or constexpr variable.
```suggestion
  constexpr int GUI_MSG_FLAG_UPDATE_LIST = 0x00000001;
```

Reply:
These are bit flags and can be used combined. Making as variable is not a convention.

Reply:
I know those are bit flags, but variables can also get bit combined.